// Initiating the testing framework

test("An initial failing test", () => {
  throw new Error('Write a proper test!')
});